near-sdk/src/types/gas.rs

```diff
@@ -20,6 +22,14 @@ use serde::{de, Deserialize, Deserializer, Serialize, Serializer};
 #[repr(transparent)]
 pub struct Gas(pub u64);
 
+pub const ONE_TGAS: Gas = Gas(u64::pow(10, 12));
```

Comment:
Maybe worth adding the const to the `Gas` impl instead of here so that you can just use `Gas::ONE_TGAS` and not have to import or know about this specifically? Just an opinion